### PR TITLE
Tidy mentor request dropdown

### PR DESCRIPTION
**Pull request by iHiD**
A few small tweaks including adding a short format to the date helper we use.

@kntsoriano There doesn't seem to be test coverage for the solution-summary in the JS tests. Is that intentional because it's all in system tests, or an oversight?

### CODE REVIEW

File: app/javascript/components/common/MentorDiscussionSummary.tsx [hunk 1]
```diff
@@ -30,7 +30,7 @@ export const MentorDiscussionSummary = ({
         />
         {postsCount}
       </div>
-      <time dateTime={createdAt}>{fromNow(createdAt)}</time>
+      <time dateTime={createdAt}>{fromNow(createdAt, true)}</time>
```
Comment (ErikSchierboom) [inline app/javascript/components/common/MentorDiscussionSummary.tsx]: I found that the boolean nature of the argument made it hard to understand what its function is. Maybe introduce a separate function: `shortFromNow` or something like that?

Comment (iHiD) [reply app/javascript/components/common/MentorDiscussionSummary.tsx]: Done. Thanks.

**Comment by iHiD** [comment]
Merging this in. @kntsoriano Please see my comment in the OP.

**Comment by kntsoriano** [comment]
@iHiD The system tests you wrote covers the tests for that component.

**Pull request merged**